.github/workflows/lint.yml

```diff
@@ -32,8 +32,18 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v2
 
+        # Use custom problem matchers for better GH annotations
+      - name: Remove old matchers
+        run: echo "::remove-matcher owner=rust::"
+
+      - name: Add problem matchers
+        run: echo "::add-matcher::.github/problemMatchers/rust.json"
```

Comment:
why was this change included?

Reply:
Problem Matchers allow for better annotations in the GitHub summary section of action runs